lib/src/date-input/DateInput.stories.tsx

```diff
@@ -118,8 +130,9 @@ const YearPicker = () => (
 );
 
 export const ShowYearPicker = YearPicker.bind({});
```

Comment:
Change the title of the story to show the behaviour that the component should have.